I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 222, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script.rpy", line 40, in script
    with Pause()
TypeError: __init__() takes at least 2 arguments (3 given)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 222, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script.rpy", line 40, in script
    with Pause()
  File "D:\Users\Wayne Hartigan\Documents\renpy-6.99.14-sdk\renpy\ast.py", line 1310, in execute
    renpy.exports.with_statement(trans, paired)
  File "D:\Users\Wayne Hartigan\Documents\renpy-6.99.14-sdk\renpy\exports.py", line 1408, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\Users\Wayne Hartigan\Documents\renpy-6.99.14-sdk\renpy\display\core.py", line 2186, in do_with
    clear=clear)
  File "D:\Users\Wayne Hartigan\Documents\renpy-6.99.14-sdk\renpy\display\core.py", line 2620, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "D:\Users\Wayne Hartigan\Documents\renpy-6.99.14-sdk\renpy\display\core.py", line 2878, in interact_core
    new_widget=layers_root)
  File "D:\Users\Wayne Hartigan\Documents\renpy-6.99.14-sdk\renpy\curry.py", line 38, in __call__
    **dict(self.kwargs.items() + kwargs.items()))
TypeError: __init__() takes at least 2 arguments (3 given)

Windows-8-6.2.9200
Ren'Py 6.99.14.3135
Script Testing 1.0
